types/sap__xsenv/index.d.ts

```diff
@@ -0,0 +1,56 @@
+// Type definitions for @sap/xsenv 1.2
+// Project: https://npm.sap.com/@sap/xsenv
+// Definitions by: [Michael Müller] <https://github.com/mad-mike>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/* tslint:disable no-single-declare-module
```

Comment:
Actually, it should be possible to do away with the `declare module`. You just need a path mapping like `google-cloud__storage` has:
```json
"paths":{
  "@sap/xsanv": ["sap__xsenv"]
},
```